from mysql import connector
import getpass
p = getpass.getpass()
try:
    mydb=connector.connect(host='localhost',user='root',passwd=p)
    print ("Connected")
except connector.DatabaseError as e:
	print("username or password is incorrect")